source/sixtrack.f90

```diff
@@ -5635,7 +5630,7 @@ subroutine corrorb
       ivflag=0
       icflag=0
 
-      write(lout,*)
+      write(lout,"(a)") ""
       write(lout,10000)
```

Comment:
What is FORMAT 10000 ? Please add a comment or make it explicit.

Reply:
There are a couple of more FORMATs in this routine, but cleaning them up should probably be done in another PR.